deployment/clouddeploy/osv-api/run.yaml

```diff
@@ -10,6 +10,9 @@ spec:
         resources:
           limits:
             memory: 4Gi
+        startupProbe:
```

Comment:
Are you happy to accept the defaults for these fields?

```
          initialDelaySeconds: DELAY
          timeoutSeconds: TIMEOUT
          failureThreshold: THRESHOLD
          periodSeconds: PERIOD
```

Reply:
Personally I'm fine with leaving them as default, since that's what we have been using before anyway.

I'm not sure if there are better values to be setting these?

Reply:
Yeah, it's something we can tune later if you don't have a strong intuition for what you would want to set these values now. We can check what they are once deployed in test and make sure it makes sense for this backend.